compiler/common-artifacts/CMakeLists.txt

```diff
@@ -15,14 +15,23 @@ endif()
 # Create python virtual environment with tensorflow 1.13.2
 set(VIRTUALENV_OVERLAY "${NNCC_OVERLAY_DIR}/venv_1_13_2")
```

Comment:
How about changing `VIRTUALENV_OVERLAY` to `VIRTUALENV_OVERLAY_TF_1_13_2` as well?

Reply:
I've updated the variable name.